README.md

```diff
@@ -41,6 +41,43 @@ bin/console analyze:php:code-compliance-report
 ​
 This command returns the content of a previously generated report. Alternatively, you can view a report via an editor.
 ​
+
+
+### Define custom prefixes for core entity names
+
+When evaluator checks project-level code entities for existing and potential matches with the core ones, it skips the entities that have the `Pyz` prefix in their name. Such are considered unique and will not conflict with core entities in future because there will never be an entity with the `Pyz` prefix in the core.
```

Comment:
`Such are` -> `Such entities are`. 

If you use `such` without a noun, you need to say `Such is`, for example `Such is life`.

Reply:
Thanks ++